README.md

```diff
@@ -1,75 +1,80 @@
-# Spyder - The Scientific PYthon Development EnviRonment
+![Spyder — The Scientific Python Development Environment](
+./img_src/spyder_readme_banner.png)
 
-Copyright © Spyder Project Contributors.
+*Copyright © 2009–2018 Spyder Project Contributors*
 
 
-## Project details
 [![license](https://img.shields.io/pypi/l/spyder.svg)](./LICENSE)
 [![pypi version](https://img.shields.io/pypi/v/spyder.svg)](https://pypi.python.org/pypi/spyder)
 [![Join the chat at https://gitter.im/spyder-ide/public](https://badges.gitter.im/spyder-ide/spyder.svg)](https://gitter.im/spyder-ide/public)
+[![OpenCollective Backers](https://opencollective.com/spyder/backers/badge.svg?color=blue)](#backers)
+[![OpenCollective Sponsors](https://opencollective.com/spyder/sponsors/badge.svg?color=blue)](#sponsors)
 
-
-## Build status
+[![CircleCI](https://circleci.com/gh/spyder-ide/spyder.svg?style=svg)](https://circleci.com/gh/spyder-ide/spyder)
```

Comment:
Please leave this one in its previous position. I'm very accustomed to this right now, sorry.

Reply:
It just looks really odd in the middle there since it is in a very different style from all the others, but its not a big deal.

Reply:
There's a better badge for CircleCI somewhere (I don't know where). You could look for it and replace it here, if you prefer.

Reply:
Thanks; done. Just needed to change a parameter in the existing link.